### ISSUE TITLE
Error in book

### ISSUE THREAD

**Issue by JaakobKind** [vasishth/bayescogsci#43]
On page 4 of the book "An Introduction to Bayesian Data Analysis for Cognitive Science", I find the statement" The probabilities of all possible events in the entire sample space must sum up to 1." 
I do not like this statement very much, because it would not even become true if "events" would be replaced by "elementary events" (because the sum of the probabilities of all elementary events is 0 for a continuous distribution).

**Comment by vasishth** [comment]
yes, you are absolutely right. I have fixed this in the next revision of the book. Do you agree with this revision? The @'s refer to bibliography references, you won't see them resolved in this source text:

> Both the frequency-based and the uncertain-belief perspective have their place in statistical inference, and depending on the situation, we are going to rely on both ways of thinking. Regardless of these differences in perspective, the probability of an outcome happening is defined to be constrained in the following way.  For now, we consider discrete outcomes, such as obtaining a 6 when tossing a six-sided die. The statements below are not formal statements of the axioms of probability theory; for more details (and more precise formulations), see @RossProb or @kolmogorov2018foundations. Another formal presentation is in @resnick2019probability.

* The probability of an outcome must lie between 0 and 1, where 0 means that the outcome is impossible and cannot happen, and 1 means that the outcome is certain to happen.
* For any two mutually exclusive outcomes, the probability that one or the other occurs is the sum of their individual probabilities.
* Two outcomes are independent if and only if the probability of both outcomes happening is equal to the product of the probabilities of each outcome happening.
* The probabilities of all possible outcomes in the entire sample space must sum up to 1.

The above definitions are based on the axiomatic definition of probability by @kolmogorov2018foundations.

**Comment by JaakobKind** [comment]
I would keep the term "event" in the first three statements, especially in the 3rd one, because outcomes cannot be independent. 
Hence, both terms "outcome" and "event" should be defined. Events are used later in Section 1.2. Hence, readers should know that they are subsets of the sample space (while outcomes are elements of the sample space). In a very strict terminology, outcomes would not have a probability, but the corresponding elementary events.

**Comment by vasishth** [comment]
i think that then i have to introduce the idea of a power set. let me think about this again.

**Comment by vasishth** [comment]
@JaakobKind I rewrote section 1.1. I think this version is accurate. Let me know if there is a mistake:

http://bruno.nicenboim.me/bayescogsci/ch-intro.html#introprob

Thanks.
